install_config/topics/ocp_osp_enabling_octavia.adoc

```diff
@@ -51,7 +51,7 @@ Hat OpenStack Platform release installed.
 ====
 
 The following step pulls the container images from registry.access.redhat.com
-to the undercloud node. This may take soem time depending on the speed of the
+to the undercloud node. This may take some time depending on the speed of the
```

Comment:
s/may/might

I know that _may_ was there already. But _might_ is the preferred term.